setup.py

```diff
@@ -88,5 +88,6 @@ def read(*filenames, **kwargs):
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Framework :: Nengo",
```

Comment:
This should be in alphabetical order, as the other classifiers are, and honestly I would leave this for a separate PR because it's not really related to the point of this one.

Reply:
I wanted to be efficient and also avoid the possibility of a one line change taking over a month to get through (e.g., https://github.com/nengo/nengo/pull/1502) so I just did it, especially since the PyPi team requested that it be "of immediate use". Accordingly I plan to do a `nengolib` release with this asap. But sure I've reverted the commit and will make a separate PR.